### PR TITLE
Project List Rewrite

### PR DESCRIPTION
**Pull request by noahmalmed**
This pull request addresses: #146 #143 #147 #131

There are a few different things happening here:
1) Added new font (Karla)
2) Update of Projects action code
3) Redesign Project List
4) Update nav bar for new style guide

# Review Checklist

- [x] Does it work in Android and iOS?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Are tests passing?

### CODE REVIEW

File: src/components/ZooWebView.js [hunk 1]
```diff
@@ -28,18 +30,29 @@ const mapDispatchToProps = (dispatch) => ({
...
+  static renderNavigationBar() {
+    return <NavBar pageKey={PAGE_NAME} showBack={true} />
+  }
+
   componentWillMount() {
     this.props.setIsFetching(true)
   }
 
+  componentDidMount() {
+    this.props.navBarActions.setTitleForPage(this.props.project.display_name, PAGE_NAME)
```
[the remark this answers is not in the document]
Comment (noahmalmed) [reply src/components/ZooWebView.js]: Addressed

**Comment by wgranger** [comment]
I get an error (red screen) whenever I click on "Recent."
screen shot 2018-03-14 at 3 50 27 pm

File: src/components/common/Separator.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const Separator = ({style}) => {
+    return (
+        <View style={[styles.defaultStyle, style]} />
+    );
+}
+
+const styles = EStyleSheet.create({
+    defaultStyle: {
+        borderBottomColor: '#cbcccb',
```
Comment (wgranger) [inline src/components/common/Separator.js]: Should this be a constant, or use one of the predefined ones in theme.js?

Comment (noahmalmed) [reply src/components/common/Separator.js]: Yeah, it looks like I use this color elsewhere, I'll pull it out

**Comment by wgranger** [comment]
Two style items need to be addressed. 

On Android, the username was covered up by the icon above. 
screen shot 2018-03-15 at 1 37 32 pm

On both Android and iOS, the tutorial would not scroll down to the bottom line. This is on the Planet Four: Ridges project.
screen shot 2018-03-15 at 1 39 19 pm

**Comment by noahmalmed** [comment]
Nice catches! Also, creepy avatar 😄. Issues addressed

**Comment by noahmalmed** [comment]
updated snapshots 🙄 not really sure I see the use in these types of unit tests

**Comment by wgranger** [comment]
Nice. Looks good. Merging.

**Pull request merged**